src/utils.ts

```diff
@@ -25,5 +27,5 @@ export function fireDeterminedAction(el: HTMLElement, path: string[]): void {
 }
 
 export function expandHotkeyToEdges(hotkey: string): string[][] {
-  return hotkey.split(',').map(edge => edge.split(' '))
+  return hotkey.split(',').map(edge => edge.split(' ').map(k => normalizeHotkey(k)))
```

Comment:
this is where the normalization is applied, so we take the user-provided hotkey string, normalize (or localize?) it for the current platform and store that on the trie. Then when we later serialize a keyboard event we compare it to the normalized version of the string.